docs/ABI.rst

```diff
@@ -1039,7 +1039,7 @@ Types
   type-list ::= list-type '_' list-type*     // list of types
   type-list ::= empty-list
 
-  list-type ::= type identifier? 'z'? 'd'?   // type with optional label, inout convention and variadic specifier
+  list-type ::= type identifier? 'z'? 'h'? 'd'?   // type with optional label, inout convention, shared convention, and variadic specifier
```

Comment:
I expect shared to be common enough to warrant abbreviation.  It'll be used heavily in the standard library if nothing else.